azurerm/resource_arm_container_group_test.go

```diff
@@ -25,6 +26,20 @@ func TestAccAzureRMContainerGroup_basicLinux(t *testing.T) {
 				Config: config,
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMContainerGroupExists("azurerm_container_group.test"),
+					resource.TestCheckResourceAttr(resourceName, "container.#", "1"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.command", "/bin/bash -c ls"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.cpu", "0.5"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.env_vars.%", "2"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.env_vars.foo", "bar"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.env_vars.foo1", "bar1"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.image", "microsoft/aci-helloworld:latest"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.memory", "0.5"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.name", "hw"),
+					resource.TestCheckResourceAttr(resourceName, "container.0.port", "80"),
+					resource.TestCheckResourceAttr(resourceName, "ip_address_type", "Public"),
+					resource.TestCheckResourceAttr(resourceName, "os_type", "Linux"),
+					resource.TestCheckResourceAttr(resourceName, "tags.%", "1"),
+					resource.TestCheckResourceAttr(resourceName, "tags.environment", "testing"),
```

Comment:
I'd suggest that checking all of these is probably overkill, since the `Import` tests will do this for us (by deploying the input configuration, and then comparing the remote state against it - without specifying each fields values)

Instead - we could probably just check the OS Type / Environment Variables here?

Reply:
In addition, I'll leave command in there too, remove the others.